src/php/eZ/Publish/Profiler/Executor.php

```diff
@@ -22,6 +22,10 @@ public function run( array $constraints, Aborter $aborter )
             {
                 $constraint->run( $this, $this->logger );
             }
+
+            // Force garbage collection between runs so we do not hit it while
+            // running an executor which will fail the performance analysis.
+            gc_collect_cycles();
```

Comment:
@beberlei OK, the comment is not entirely clear: Run it because we do not want GC side effects from an earlier run. If the Execution in between triggers a GC run then we want it there and want to know about it. Might not be entirely true, but this is to at least reduce GC side effects…